ql/src/java/org/apache/hadoop/hive/ql/plan/ImportTableDesc.java

```diff
@@ -332,4 +330,13 @@ public String getDatabaseName() {
   public boolean isMaterializedView() {
     return table.isMaterializedView();
   }
+
+  public TableType tableType() {
```

Comment:
we are providing additional info that might not b useful and that is leading to code change which might be not required,